utils/tang/files/tang.init

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh /etc/rc.common
+
+START=50
+USE_PROCD=1
+
+start_service() {
+	KEYS=$(find /usr/share/tang/db -name "*.jw*" -maxdepth 1 | wc -l)
+	if [ -z "${KEYS}" ] || [ "${KEYS}" = "0" ]; then # if db is empty generate new key pair
+		mkdir -p /usr/share/tang/db
+		/usr/sbin/tangd-keygen /usr/share/tang/db
+	fi
+
+	config_load "tang"
+	config_get_bool enabled "config" 0
+	config_get port $1 port "9090"
```

Comment:
```suggestion
	config_get_bool enabled "config" "enabled" 0
	config_get port "config" "port" "9090"
```

`config_get <var name> <uci section> <uci option> [<fallback val>]`

`$1` doesn't seem to exist here? ;)

Reply:
Great catch, thank you!